### ISSUE TITLE
[Feature Request] Favoriting/Hiding Notebooks

### ISSUE THREAD

**Issue by zachkirsch** [HappenApps/Quiver#208]
It would be great if you could hide notebooks - I have a bunch of notebooks from previous classes that I barely look at (but still do sometimes). It would be great if I could hide my old notebooks, favorite my current notebooks (so they show at the top), or at least be able to manually reorder the list of notebooks in the sidebar.

**Comment by ylian** [comment]
This will be solved by implementing nested notebooks: https://github.com/HappenApps/Quiver/issues/2

**Comment by zachkirsch** [comment]
Yeah I think that I could do that with nested notebooks. The only drawback
is that I would have to group a bunch of very unrelated notebooks into one
big dummy notebook, right?

**Comment by ylian** [comment]
Manual reordering of notebooks will be implemented too.
